packages/satellite/rubygem-foreman_theme_satellite/rubygem-foreman_theme_satellite.spec

```diff
@@ -0,0 +1,93 @@
+# template: foreman_plugin
+%global gem_name foreman_theme_satellite
+%global plugin_name theme_satellite
+%global foreman_min_version 3.7
+
+Name: rubygem-%{gem_name}
+Version: 13.1.0
+Release: 1%{?foremandist}%{?dist}
+Summary: This is a plugin that enables building a theme for Foreman
+License: GPLv3
+URL: https://github.com/RedHatSatellite/foreman_theme_satellite
+Source0: https://rubygems.org/downloads/%{gem_name}-%{version}.gem
+
+# start specfile generated dependencies
+Requires: foreman >= %{foreman_min_version}
+Requires: ruby >= 2.7
+BuildRequires: foreman-assets >= %{foreman_min_version}
+BuildRequires: foreman-plugin >= %{foreman_min_version}
+Requires: ruby >= 2.7
```

Comment:
I wonder why ruby is listed twice.

Reply:
solid q, I have no idea why the template did it like this.

Reply:
https://github.com/theforeman/foreman-packaging/blob/0959e5e0625daa62f6693c91ba27961cb240a90f/gem2rpm/foreman_plugin.spec.erb#L46-L48

https://github.com/theforeman/foreman-packaging/blob/0959e5e0625daa62f6693c91ba27961cb240a90f/gem2rpm/foreman_plugin.spec.erb#L56-L58

I think it was added here https://github.com/theforeman/foreman-packaging/pull/8079/files#diff-b2bcfeb556d566a4b0e36677afbe6912487b4aecf0a06416a72242fcd7827d99

Reply:
https://github.com/theforeman/foreman-packaging/pull/9916